### ISSUE TITLE
Student grade calculator

### ISSUE THREAD

**Issue by MohitNamdev22** [jay-2000/Open-source-hacktoberfest#49]
Can I add a project which can find the grade of student by taking input of marks.

**Comment by jay-2000** [comment]
> Can I add a project which can find the grade of student by taking input of marks.

yeah if its in website form

**Comment by MohitNamdev22** [comment]
Yeah it will be in html, css, js only

**Comment by jay-2000** [comment]
> Yeah it will be in html, css, js only

yeah then go ahead @MohitNamdev22 i'll assign it to you under hacktoberfest

**Comment by jay-2000** [comment]
Don't forget to mention screenshot of your project while opening/submitting the PR @MohitNamdev22

**Comment by MohitNamdev22** [comment]
> MohitNamdev22

Yeah
